sled-agent/src/opte/illumos/port_manager.rs

```diff
@@ -267,7 +267,7 @@ impl PortManager {
         // The Port object's drop implementation will clean up both of those, if
         // any of the remaining fallible operations fail.
         let port_name = self.inner.next_port_name();
-        let hdl = OpteHdl::open(OpteHdl::DLD_CTL)?;
+        let hdl = OpteHdl::open(OpteHdl::XDE_CTL)?;
```

Comment:
So, this isn't really part of this PR, but I've always found it a bit odd that consumers of `opte-ioctl` need to know anything about `XDE_CTL` or `DLD_CTL` at all. What do you think about making this API work like so?

```rust
let hdl = OpteHdl::open()?;
```

I.e., we use `XDE_CTL` internally, rather than having it part of the API.

Reply:
Opened https://github.com/oxidecomputer/opte/issues/341